packages/websocket/websocket.0.9.2/files/websocket.install

```diff
@@ -0,0 +1,12 @@
+bin: [
+  "_build/tests/wscat.native" {"wscat"} ]
+lib: [
+  "_build/lib/websocket.a" {"websocket.a"}
+  "_build/lib/websocket.cma" {"websocket.cma"}
+  "_build/lib/websocket.cmi" {"websocket.cmi"}
+  "?_build/lib/websocket.cmti" {"websocket.cmti"}
+  "_build/lib/websocket.cmx" {"websocket.cmx"}
+  "_build/lib/websocket.cmxa" {"websocket.cmxa"}
+  "_build/lib/websocket.cmxs" {"websocket.cmxs"}
+  "_build/lib/websocket.mli" {"websocket.mli"}
+  "_build/pkg/META" {"META"} ]
```

Comment:
in the future, could you put this file in the repository? It makes pinning easier.  Also, the `cmx/a/s` files can also be optional.